### DISCUSSION TITLE
Emails from supabase add a '3D' right before the value

### DISCUSSION THREAD

**Discussion by stretch07** [supabase/supabase#5295] [Questions] [answered]
Take a look at this:
![image](https://user-images.githubusercontent.com/78528552/152258244-81fdf4f4-7fc2-43c8-a773-41c719ba2c9a.png)
I did not add those 3d's behind my apostrophes. Is this a security feature, or a bug?

**Comment by GaryAustin1** [comment, marked as answer]
https://stackoverflow.com/questions/4016067/whats-a-3d-doing-in-this-html
